openmetadata-ui/src/main/resources/ui/src/components/DataAssets/AssetsSelectionModal/useAssetSelectionContent.tsx

```diff
@@ -376,6 +374,7 @@ export const useAssetSelectionContent = ({
           quickFilterQuery as QueryFilterInterface
         );
 
+        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
Can we remove this comment? We can use if statement to resolve the eslint error here